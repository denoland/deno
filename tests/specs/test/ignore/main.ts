for (let i = 0; i < 5; i++) {
  Deno.test({
    name: `test ${i}`,
    ignore: true,
    fn() {
      throw new Error("unreachable");
    },
  });
}
for (let i = 5; i < 10; i++) {
  Deno.test.ignore({
    name: `test ${i}`,
    fn() {
      throw new Error("unreachable");
    },
  });
}
