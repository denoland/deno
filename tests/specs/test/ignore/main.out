Check [WILDCARD]/main.ts
running 10 tests from ./main.ts
test 0 ... ignored ([WILDCARD])
test 1 ... ignored ([WILDCARD])
test 2 ... ignored ([WILDCARD])
test 3 ... ignored ([WILDCARD])
test 4 ... ignored ([WILDCARD])
test 5 ... ignored ([WILDCARD])
test 6 ... ignored ([WILDCARD])
test 7 ... ignored ([WILDCARD])
test 8 ... ignored ([WILDCARD])
test 9 ... ignored ([WILDCARD])

ok | 0 passed | 0 failed | 10 ignored ([WILDCARD])

